README.md

```diff
@@ -25,7 +30,11 @@ The Manager contains the UI and API to support it - the API will create a Config
 The Watcher will be notified of the presence of the flow `ConfigMap` and will deploy the necessary components (DeploymentConfigs and ConfigMaps).
 
 The Manager can be run outside OpenShift using `mvn clean package wildfly-swarm:run`.
-The Watch can be run outside OpenShift using `mvn clean package; java -jar target/FlowController.jar`.
+The Watcher can be run outside OpenShift using `mvn clean package; java -jar target/FlowController.jar`.
```

Comment:
I was wondering that - does it really make sense to run it outside of k8s / Openshift ?

Reply:
Running outside the OpenShift instance that it's managing will be necessary for multi-cloud support. The Watcher might be either completely outside or in a different OpenShift instance

Reply:
ah, ok - thanks